src/askui/models/shared/settings.py

```diff
@@ -0,0 +1,71 @@
+from pydantic import BaseModel, Field
+
+
+class ChatCompletionsCreateSettings(BaseModel):
+    """
+    Settings for creating chat completions.
```

Comment:
I think in the future it would also be fine to simply include links to where the user can find documentation instead of having to document everything ourselves. But this here has also advantages. WDYT?